proc_IR.py

```diff
@@ -1,26 +1,30 @@
 from pyspecdata import *
 from scipy.optimize import leastsq,minimize
-from proc_scripts import hermitian_function_test, zeroth_order_ph, recovery, integrate_limits, correl_align, ph1_real_Abs, postproc_dict,DCCT
+from proc_scripts import hermitian_function_test, zeroth_order_ph, recovery, integrate_limits, correl_align, ph1_real_Abs, postproc_dict,DCCT,integral_w_errors
 from sympy import symbols, latex, Symbol
 from matplotlib import *
 from scipy.signal import tukey
 import matplotlib.pyplot as plt
 import numpy as np
 from sympy import exp as s_exp
-init_logging('debug')
-fl = figlist_var()
-t2 = symbols('t2')
-# {{{ input parameters
 def select_pathway(s,pathway):
     retval = s
     for k,v in pathway.items():
         retval = retval[k,v]
     return retval
+init_logging('debug')
+fl = figlist_var()
+t2 = symbols('t2')
+# {{{ input parameters
+this_l = 0.032 # pick number in l curve right before it curves up
+l = sqrt(np.logspace(-8.0,0.5,35)) # play around with the first two numbers to get good l curve,number in middle is how high the points start(at 5 it starts in hundreds.)
```

Comment:
OK, so here, I will admit that these need to be hard-coded,  but they should be **at the top**, so that it's clear that they are adjustable parameters!